example/test.py

```diff
@@ -421,12 +415,12 @@ def test_expected_processes(self):
              '--concurrency 1'],
         ])))
 
-    def test_amqp_queues_created(self):
+    def test_amqp_queues_created(self, amqp_container):
```

Comment:
Do we not need `worker_container` here as well so we can be sure the startup has completed before this test runs?